mood-analysis.rb

```diff
@@ -28,5 +34,62 @@ def analyze_mood(words)
   "05/11 Yay, yay, yay! I'm having a awfuly great day."
 ]
 
-puts analyze_mood(text[0])
-puts analyze_mood(text[1])
+# print out date + mood by calling analyze mood
+def week_moods(array)
+  array.each do |day|
+    date = day[0..4]
+    puts "#{date} #{analyze_mood(day)}"
+  end
+end
+
+def happy_days(array)
+  happy = 0
+  days = 0
+  array.each do |day|
+    mood = analyze_mood(day)[-3..-1]
```

Comment:
I'm not sure what the [-3..-1] is for or why it's here. Could look it up, but trying to churn through 24 PRs.